README.md

```diff
@@ -17,21 +17,3 @@ In case...
 - ...or having great ideas for new content
 
 ... please head over to the issues section and [report an issue](https://github.com/royalapplications/toolbox/issues). Support is provided on a best-effort basis from us and the original script authors.
-
```

Comment:
@eiabea Please add a link to CONTRIBUTING.md in the readme.

Reply:
Added in e97292ff2b92d1b114e427cc8a8cbccfb5af314e